Utilities/DataFlow/doc/SubframeBuilderDevice.1.in

```diff
@@ -0,0 +1,48 @@
+.\" Manpage for SubframeBuilderDevice.
+.TH man 1 "12 May 2017" "1.0" "SubframeBuilderDevice man page"
```

Comment:
I have started to use AliceO2 instead of man at this point to mark the AliceO2 related man pages.